distribution/packages/build.gradle

```diff
@@ -311,12 +312,19 @@ Closure commonDebConfig(boolean oss, boolean jdk) {
 
     version = project.version.replace('-', '~')
     packageGroup 'web'
-    requires 'bash'
+
+    // versions found on oldest supported distro, centos-6
+    // nocommit: why are bash and libc only required for deb?
```

Comment:
The RPM config requires `/bin/bash`.

Reply:
`libc` needs to be addressed indeed though.

Reply:
I did some more research and found rpm automatically finds lib deps based on binary paths, eg for libc being needed by /bin/bash, so that is why it is not needed in rpm as a direct dependency.